README.md

```diff
@@ -242,11 +242,21 @@ info-type logs.)
 - Constant keys are generally a good idea.
 - Be consistent across your codebase.
 - Keys should naturally match parts of the message string.
+- Use lower case.
+- Separate multiple words with hyphens ("some-complex-key") to enhance
+  readability if necessary.
```

Comment:
I just found that Kubernetes has already decided to use lowerCamelCase: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments

I don't like it and still think that hyphens would be better, but it is a bit too late now for that. Instead of suggesting some different approach, logr should adopt the Kubernetes approach. I don't feel strongly enough about this to propose rewriting existing log messages in Kubernetes with hyphens...

Reply:
The point about hyphens not being typical symbols in languages is good, and underscores are hard to type on some non-english keyboards (so I hear?).  If we want "simple" names like "error" and "level" then the best option probably is `lowerCamelCase` (or else we get "Error" and "Lvl").